### ISSUE TITLE
Cross compilation page corrections / improvements

### ISSUE THREAD

**Issue by mewalig** [jqlang/jq#1007]
Couple corrections/clarifications to the cross compilation wiki (https://github.com/stedolan/jq/wiki/Cross-compilation), at least for building the Win64 static lib binaries from OSX:
- the prefix should not end with "lib" in the following line:

```
./configure --target=win64-x86_64 --host=x86_64-w64-mingw32 --prefix=/home/build/proto/win64/usr/local/lib
```
- The Makefile that is created in build/win64/tmp needs a couple of tweaks:
  - DEFAULT_INCLUDES needs to also have -I/path/to/build/win64/tmp/src in order to find version.h
  - libjq_la_LIBADD needs to also have -lshlwapi in order to find PathIsRelativeA

Once those were addressed the build worked, at least for a static lib

**Comment by nicowilliams** [comment]
The first part you can correct yourself: the wiki is open to edits by any and all github users.

The second part I have to fix.

**Comment by itchyny** [comment]
This issue seems to have been resolved by #1073 in 2016, so closing.
